### ISSUE TITLE
Replace commited region file with structure magic.

### ISSUE THREAD

**Issue by Possseidon** [Possseidon/the-depths#1]
Currently, everytime the testing area in the map changes, 4 mb of git-lfs get added to the repo, which isn't too bad, but still doesn't make me super happy either.

A reasonable solution would be to simply not commmit any `.mca` file at all and using `.mcfunctions` to:

1. Remove the spawn platform.
2. Generate map area floor and barrier ceiling.
3. Generate the testing area from a few of structure files.

This way, the testing area can also be moved closer to the map area, as they don't need to be in separate region files anymore.

Once finished, it's probably smart to get rid of the existing region files from the main branch by rewriting the history a bit to save on those few mb.

To sum it up:

- [x] Remove `.mca` file and replace it with `.mcfunction` that somehow get triggered once when to world loads initially.
- [x] Inlcude "remove spawn platform" and "generate map floor and barrier ceiling".
- [x] Remove `.mca` from the git history.
